docs/building/forms.md

```diff
@@ -21,36 +21,61 @@ Form
 FormMeta
     completion_status_show      - (bool) indicates a form completion status
     completion_status_in_header - (bool) indicates the status in the header
-    confirmation_needed         - (bool) shows e menu at the end of the form
-                                         so user can confirm the choices
+    confirmation_needed         - (bool) shows a menu at the end of the form so user can confirm the choices
     
 FormItem
-    type           - (string) enum: "string", "int", "float", "date", "datetime"
-        "string"   - the user should enter a string during this step
-        "int"      - the user should enter a valid number
-        "float"    - the user could enter a floating number
-        "date"     - the user should enter a date
-        "datetime" - the user should enter a date and time
-
-    name        - (string) the name of this FormItem, used in form serialization
-    description - (string) the description of this FormItem
-    header      - (string) if provided will overwrite the Form.header
-    footer      - (string) if provided will overwrite the Form.footer
-
-FormItemMenu
-    type        - (string) defaults to "form-menu"
-    name        - (string) the name of this FormItemMenu, used in form serialization
-    header      - (string) if provided will overwrite the Form.header
-    footer      - (string) if provided will overwrite the Form.footer
-    body        - (array) FormItemMenuItem objects - see below
-
-
-FormItemMenuItem
-    type        - (string) defaults to "menu"
-    value       - (string) the value of this FormItemMenuItem, used in form
-                           serialization
+    type - (string) enum: "string", "hidden", "int", "float", "date", "datetime", "form-menu"
+        "string"    - the user should enter a string during this step
+        "hidden"    - will not be displayed to the user
+        "int"       - the user should enter a valid number
+        "float"     - the user could enter a floating number
+        "date"      - the user should enter a date
+        "datetime"  - the user should enter a date and time
+        "form-menu" - the user should choose an option from the menu
+
+    name                       - (string) the name of this FormItem, used in form serialization
+
+    body                       - (array of objects) MenuItemFormItem objects - see below - required only for type=form-menu
+    chunking_footer            - (string) shown in the footer of the sms chunks
+    confirmation_label         - (string) shown in the confirmation menu
+    description                - (string) the description of this FormItem
+    editable                   - (bool) indicates this form item is editable or not
```

Comment:
From my point of view this field should be the reverse of hidden. I mean, a hidden item should be not editable. Too many options lead to even more combinations of them that can be hard to explain or to be understood. What do you think?

Reply:
Yes, you are right. It's redundant https://github.com/ONEmCommunications/nautilus/commit/9e40cce93#diff-e31d0443f9ab445b7b94b64ed3b628a2R320

Reply:
Probably better to just take it out ..